src/log.js

```diff
@@ -181,7 +181,7 @@ class Log extends GSet {
     return this._entryIndex[entry.cid || entry] !== undefined
   }
 
-  traverse (rootEntries, amount = -1) {
+  traverse (rootEntries, amount = -1, endHash) {
```

Comment:
Does the result include the `endHash` entry?

Reply:
Yes it does